### DISCUSSION TITLE
Show a dropdown to change refresh rate in Exclusive Fullscreen.

### DISCUSSION THREAD

**Discussion by dumbuz** [ppy/osu#23132] [Ideas] [answered]
I was messing around with refresh rates on my monitor, but there's some friction between changing refresh rates on osu!lazer. First, you have to change your refresh rate in system settings, then change osu! to Borderless, then to Fullscreen again. This especially makes things hard for people who play osu! at a different refresh rate than their system (for ex: a monitor that overclocks to 165hz but user only uses overclock in osu! and uses 144hz elsewhere)

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue in https://github.com/ppy/osu/issues/3879.
